### PR TITLE
github: use the same issue templates as those found in LND repository

### PR DESCRIPTION
**Pull request by ffranr**
Updates issue templates so that they match those found in the LND repository.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+contact_links:
+  - name: Discussions
+    url: https://github.com/lightninglabs/taproot-assets/discussions
+    about: For general or troubleshooting questions or if you're not sure what issue type to pick.
+  - name: Documentation for taproot-assets
+    url: https://docs.lightning.engineering/the-lightning-network/taproot-assets
+    about: Please make sure the documentation cannot answer your question first.
+  - name: Lightning Community Slack
+    url: https://lightning.engineering/slack.html
+    about: Please ask and answer questions here.
```
Comment (guggero) [inline .github/ISSUE_TEMPLATE/config.yml:11]: Maybe explicitly mention the `#taproot-assets` channel here to avoid everyone posting in `#general`?

**Review by guggero — APPROVED** [review]
Nice, LGTM :tada:

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (dstadulis) [inline .github/ISSUE_TEMPLATE/config.yml:11]:
```suggestion
    about: Ask and answer questions in #taproot-assets
```

**Review by dstadulis — CHANGES_REQUESTED** [review]

**Review by jharveyb — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**